app/models/candidate_interface/gcse_qualification_details_form.rb

```diff
@@ -0,0 +1,15 @@
+module CandidateInterface
+  class GcseQualificationDetailsForm
+    include ActiveModel::Model
+
+    attr_accessor :grade, :award_year
+    validates :grade, :award_year, presence: true
+
+    def save_base(qualification)
+      return false unless valid?
+
+      qualification.update!(grade: grade, award_year: award_year)
+      qualification
```

Comment:
I don't think we need `qualification` at the end here as `#update` returns the updated object.

Reply:
removed